### ISSUE TITLE
Xamarin forms entry focused event

### ISSUE THREAD

**Issue by kkrzysiek** [xamarin/Xamarin.Forms#4897]
### Description
Cursor blinking in the Entry control is causing the Focused event to be fired on every blink. 
### Steps to Reproduce

1. Create simple project
2. Add Entry control
3. Create event for Focused: 
TestEntry.Focused += (object sender, FocusEventArgs e) =>{};
4. Run the app and click into the Entry
5. The code will be fired on every blink
### Expected Behavior
I think that it should not fire event on blinking of the cursor.
### Actual Behavior
Every cursor blink is firing the event
- Version with issue: UWP. I tested with version 3.1 and it has the same issue

### Reproduction
[EntryFocused.zip](https://github.com/xamarin/Xamarin.Forms/files/2728053/EntryFocused.zip)

**Comment by PureWeen** [comment]
@kkrzysiek  this looks like it's only an issue with UWP
I've attached a repro in the original description. Let me know if my testing seems incorrect

**Comment by bmacombe** [comment]
@PureWeen @kkrzysiek 

I decided to take a look at this and I'm wondering how you verified the focused fired was being called multiple times?.  I think that it happens when the entry is the root item on the page and you put a break point in the Focused handler, VS becomes the active Window when it hits.  When you hit F5 and move on with execution the app becomes the active window again and since the entry is the root item in the page it get's it Focus event fired, since the app just got focus again.

At least that is what I think is happening.  When I setup the Focused method to just update the page title with a count, it only seems to fired initially and then not on every cursor blink.  But if I put a break point in it will be fired every time.  I suspect this behavior is by design, I'll try to take a look at the ContentPage source to verify.

I've attached an updated sample project that shows the event is not fired on blinks if no break point is set.

If you agree this behavior is correct, this issue can probably be closed,

[EntryFocused Updated.zip](https://github.com/xamarin/Xamarin.Forms/files/3722125/EntryFocused.Updated.zip)
